apis/core/v1beta1/openstackcontrolplane_types.go

```diff
@@ -126,6 +127,11 @@ type OpenStackControlPlaneSpec struct {
 	Horizon HorizonSection `json:"horizon,omitempty"`
 
 	// +kubebuilder:validation:Optional
+	// Telemetry - Parameters related to the OpenStack Telemetry services
+	Telemetry TelemetrySection `json:"telemetry,omitempty"`
+
+	// +kubebuilder:validation:Optional
+	//+operator-sdk:csv:customresourcedefinitions:type=spec
```

Comment:
I'm not sure why this `//+operator-sdk:csv:customresourcedefinitions:type=spec` has been added as an annotation for `ExtraMounts`

Reply:
Hmm, I must have overseen this during one of rebases I did. Will remove.